please

```diff
@@ -61,9 +59,20 @@ if [ "$USE_NIX" = "1" ]; then
 
 else
   if [ "$USE_DOCKER" = "1" ]; then
+    DOCKER_HASH="`pwd``git rev-parse --abbrev-ref HEAD`"
+    DOCKER_NAME="mozilla-releng-services-v`cat ./VERSION`-`echo -n $DOCKER_HASH | sha256sum | cut -d ' ' -f 1`"
     if [ ! "$(docker ps -q -f name=$DOCKER_NAME)" ]; then
       if [ ! "$(docker ps -qa -f name=$DOCKER_NAME)" ]; then
-        docker run --privileged -td --name $DOCKER_NAME --volume="$(pwd)":/app --workdir=/app -p 7000:7000 -p 8000-8100:8000-8100 mozillareleng/services:base-`cat ./VERSION`
+        docker run \
+            --privileged \
+            -td \
+            --name $DOCKER_NAME \
+            --volume="$(pwd)":/app \
+            --volume=/var/run/docker.sock:/var/run/docker.sock \
+            --workdir=/app \
+            -p 7000:7000 \
+            -p 8000-8100:8000-8100 \
```

Comment:
Ideally these ports should be configurable. We don't necessarily need open ports for some projects (static analysis, code coverage, ...) and certainly not 100

Reply:
probably we can easily go down with open ports, but at this point i don't have the information which ports i need to open. Since this is not changing anything (except adding 7000 port for releng-docs) I would fix this in a separate PR.